### PR TITLE
Update 06-media-communication.md

### PR DESCRIPTION
**Pull request by namvdo**

### CODE REVIEW

File: content/docs/06-media-communication.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ A trivial congestion control algrorithm to estimate the incoming bitrate on the
 
 ## Generating a Bandwidth Estimate
 Now that we have information around the state of the network we can make estimates around the bandwidth available. In 2012 the IETF started the RMCAT (RTP Media Congestion Avoidance Techniques) working group.
-This working group contains multiple submitted standards for congestion control algorithms. Before then Congestion Controllers algorithms were proprietary.
+This working group contains multiple submitted standards for congestion control algorithms. Before that Congestion Controllers algorithms were proprietary.
```
Comment (mogren) [inline content/docs/06-media-communication.md:253]: I think this section needs even more fixing... 🤔 Maybe mention some of the standards? 

Maybe "Before then..." -> "Before these standards were submitted, most..." (or all?).

Comment (namvdo) [reply content/docs/06-media-communication.md:253]: Yeah, it looks clear if you rephrase like this.

Comment (mogren) [reply content/docs/06-media-communication.md:253]: The paragraph currently says:

> In 2012 the IETF started the RMCAT (RTP Media Congestion Avoidance Techniques) working group. This working group contains multiple submitted standards for congestion control algorithms. Before then, all Congestion Controllers algorithms were proprietary.

It still doesn't feel completely right, but I'm not sure how to rephrase it.

Comment (namvdo) [reply content/docs/06-media-communication.md:253]: Yeah, you can close my pull request.

**Pull request closed without merging**